insights/parsers/sssd_logs.py

```diff
@@ -44,58 +44,63 @@ class SSSDLog(LogFileOutput):
     Examples:
 
         >>> logs = shared[SSSDLog]
-
         >>> hups = logs.get("SIGHUP")
```

Comment:
Maybe this line-47 should change to    
`hups = logs.get_parsed_line("SIGHUP")`